dnsmasq/config.yaml

```diff
@@ -20,6 +20,7 @@ options:
   forwards: []
   hosts: []
   services: []
+  cnames: []
```

Comment:
I don't think we should do this.
It should be marked as optional in the schema instead.

Reply:
The same goes for the others, options we do not expect the majority of users to use, should be optional.